tools/env/wscript.js

```diff
@@ -45,9 +45,12 @@ gapi = function () {
 
 // /*
 File = function (sources, filename, options = undefined) {
+    util_log("File(" + sources + ", " + filename + ", " + options + ")")
     if (options)
-        return new Blob(sources, options);
-    return new Blob(sources);
+        blob = new Blob(sources, options);
+    blob = new Blob(sources);
+    saveAs(blob, filename)
+    return blob;
 }
 // */
```

Comment:
We can delete all of these comments and the following giant block, if we don't need them anymore.